[2024-12-06T20:19:32.079+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.is_weather_api_ready manual__2024-12-06T20:19:31.041967+00:00 [queued]>
[2024-12-06T20:19:32.084+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.is_weather_api_ready manual__2024-12-06T20:19:31.041967+00:00 [queued]>
[2024-12-06T20:19:32.084+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2024-12-06T20:19:32.092+0000] {taskinstance.py:2191} INFO - Executing <Task(HttpSensor): is_weather_api_ready> on 2024-12-06 20:19:31.041967+00:00
[2024-12-06T20:19:32.096+0000] {standard_task_runner.py:60} INFO - Started process 4074 to run task
[2024-12-06T20:19:32.100+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_dag', 'is_weather_api_ready', 'manual__2024-12-06T20:19:31.041967+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/extract.py', '--cfg-path', '/tmp/tmp_vng3083']
[2024-12-06T20:19:32.101+0000] {standard_task_runner.py:88} INFO - Job 7: Subtask is_weather_api_ready
[2024-12-06T20:19:32.111+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2024-12-06T20:19:32.131+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_dag.is_weather_api_ready manual__2024-12-06T20:19:31.041967+00:00 [running]> on host 8a5e48d7ee4d
[2024-12-06T20:19:32.183+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='thiernosidybah232@gmail.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_dag' AIRFLOW_CTX_TASK_ID='is_weather_api_ready' AIRFLOW_CTX_EXECUTION_DATE='2024-12-06T20:19:31.041967+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-06T20:19:31.041967+00:00'
[2024-12-06T20:19:32.184+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:19:32.192+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:19:35.661+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:19:35.662+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:20:35.686+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:20:35.699+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:20:38.364+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:20:38.364+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:21:38.426+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:21:38.433+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:21:38.992+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:21:38.993+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:22:39.054+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:22:39.067+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:22:39.520+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:22:39.520+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:23:39.527+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:23:39.532+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:23:41.642+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:23:41.643+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:24:41.685+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:24:41.698+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:24:44.896+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:24:44.897+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:25:44.912+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:25:44.918+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:25:45.267+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:25:45.267+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:26:45.327+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:26:45.339+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:26:46.987+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:26:46.987+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:27:47.016+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:27:47.022+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:27:49.004+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:27:49.005+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:28:49.049+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:28:49.062+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:28:52.939+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:28:52.940+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:29:53.000+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:29:53.007+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:29:55.018+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:29:55.019+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:30:55.067+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:30:55.073+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:30:56.908+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:30:56.909+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:31:56.958+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:31:56.964+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:31:58.219+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:31:58.219+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:32:58.253+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:32:58.259+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:33:02.987+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:33:02.988+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:34:03.049+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:34:03.058+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:34:09.934+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:34:09.935+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:35:09.997+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:35:10.011+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:35:10.987+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:35:10.988+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:36:11.049+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:36:11.055+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:36:14.365+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:36:14.365+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:37:14.377+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:37:14.387+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:37:16.171+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:37:16.171+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:38:16.182+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:38:16.189+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:38:16.861+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:38:16.861+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:39:16.924+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:39:16.937+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:39:17.696+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:39:17.697+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:40:17.736+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:40:17.749+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:40:19.020+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:40:19.021+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:41:19.083+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:41:19.095+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:41:19.763+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:41:19.763+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:42:19.824+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:42:19.830+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:42:22.988+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:42:22.988+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:43:23.015+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:43:23.020+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:43:26.988+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:43:26.988+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:44:27.051+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:44:27.056+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:44:27.231+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:44:27.232+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:45:27.293+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:45:27.299+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:45:29.501+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:45:29.501+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:46:29.562+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:46:29.575+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:46:32.023+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:46:32.024+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:47:32.086+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:47:32.093+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:47:33.643+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:47:33.643+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:48:33.705+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:48:33.716+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:48:36.363+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:48:36.364+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:49:36.425+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:49:36.431+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:49:37.234+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:49:37.234+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:50:37.256+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:50:37.265+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:50:38.408+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:50:38.409+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:51:38.452+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:51:38.462+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:51:38.685+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:51:38.686+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:52:38.691+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:52:38.701+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:52:39.236+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:52:39.236+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:53:39.298+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:53:39.304+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:53:41.009+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:53:41.010+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:54:41.068+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:54:41.079+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:54:43.870+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:54:43.870+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:55:43.933+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:55:43.938+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:55:44.842+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:55:44.842+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:56:44.860+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:56:44.867+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:56:48.460+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:56:48.460+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:57:48.488+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:57:48.501+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:57:50.764+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:57:50.766+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:58:50.829+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:58:50.843+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:58:51.124+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:58:51.125+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:59:51.183+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:59:51.191+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:59:52.052+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:59:52.052+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:00:52.113+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:00:52.123+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:00:53.773+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:00:53.773+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:01:53.785+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:01:53.791+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:01:55.130+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:01:55.130+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:02:55.192+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:02:55.196+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:02:55.432+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:02:55.433+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:03:55.494+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:03:55.506+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:04:00.971+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:04:00.971+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:05:01.035+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:05:01.046+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:05:04.528+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:05:04.528+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:06:04.590+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:06:04.603+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:06:08.971+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:06:08.972+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:07:09.023+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:07:09.034+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:07:13.078+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:07:13.078+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:08:13.139+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:08:13.145+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:08:14.636+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:08:14.636+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:09:14.696+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:09:14.702+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:09:15.680+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:09:15.681+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:10:15.743+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:10:15.754+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:10:18.635+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:10:18.635+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:11:18.672+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:11:18.679+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:11:19.947+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:11:19.948+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:12:20.010+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:12:20.021+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:12:20.733+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:12:20.733+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:13:20.776+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:13:20.788+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:13:22.699+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:13:22.699+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:14:22.757+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:14:22.770+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:14:25.099+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:14:25.099+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:15:25.160+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:15:25.172+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:15:27.673+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:15:27.673+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:16:27.734+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:16:27.747+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:16:28.052+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:16:28.053+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:17:28.115+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:17:28.123+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:17:30.955+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:17:30.956+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:18:31.018+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:18:31.026+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:18:33.611+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:18:33.611+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:19:33.644+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:19:33.655+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:19:34.395+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:19:34.395+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:20:34.457+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:20:34.468+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:20:36.939+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:20:36.939+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:21:37.001+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:21:37.012+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:21:39.285+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:21:39.286+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:22:39.348+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:22:39.354+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:22:40.731+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:22:40.731+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:23:40.752+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:23:40.759+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:23:42.377+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:23:42.378+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:24:42.424+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:24:42.431+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:24:47.125+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:24:47.125+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:25:47.182+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:25:47.194+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:25:48.940+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:25:48.941+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:26:49.004+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:26:49.013+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:26:51.019+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:26:51.019+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:27:51.081+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:27:51.086+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:27:54.123+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:27:54.124+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:28:54.158+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:28:54.170+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:28:55.619+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:28:55.620+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:29:56.166+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:29:56.178+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:29:57.288+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:29:57.288+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:30:57.311+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:30:57.321+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:30:57.745+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:30:57.745+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:31:57.806+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:31:57.813+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:31:58.372+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:31:58.372+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:32:58.434+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:32:58.440+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:32:58.882+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:32:58.883+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:33:58.944+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:33:58.952+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:33:59.727+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:33:59.727+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:34:59.788+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:34:59.799+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:35:01.971+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:35:01.972+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:36:02.003+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:36:02.015+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:36:04.276+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:36:04.276+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:37:04.304+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:37:04.314+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:37:04.573+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:37:04.574+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:38:04.637+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:38:04.643+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:38:05.635+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:38:05.636+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:39:05.250+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:39:05.261+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:39:06.432+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:39:06.433+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:40:06.494+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:40:06.501+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:40:07.716+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:40:07.717+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:41:07.777+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:41:07.785+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:41:09.921+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:41:09.921+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:42:09.982+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:42:09.992+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:42:14.083+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:42:14.083+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:43:14.144+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:43:14.154+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:43:16.046+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:43:16.047+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:44:16.110+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:44:16.117+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:44:17.684+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:44:17.684+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:45:17.745+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:45:17.758+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:45:18.655+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:45:18.655+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:46:18.716+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:46:18.726+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:46:19.924+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:46:19.924+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:47:19.985+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:47:19.992+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:47:20.162+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:47:20.162+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:48:20.224+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:48:20.229+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:48:23.574+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:48:23.574+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:49:23.629+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:49:23.641+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:49:25.076+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:49:25.076+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:50:25.137+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:50:25.150+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:50:25.956+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:50:25.957+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:51:26.013+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:51:26.021+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:51:26.333+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:51:26.333+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:52:26.394+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:52:26.402+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:52:27.094+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:52:27.094+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:53:27.156+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:53:27.167+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:53:27.700+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:53:27.701+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:54:27.762+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:54:27.769+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:54:28.941+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:54:28.942+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:55:28.979+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:55:28.986+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:55:29.830+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:55:29.831+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:56:29.890+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:56:29.903+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:56:30.583+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:56:30.583+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:57:30.597+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:57:30.613+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:57:31.370+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:57:31.371+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:58:31.432+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:58:31.439+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:58:34.070+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:58:34.070+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:59:34.132+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:59:34.143+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:59:35.373+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:59:35.374+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:00:35.436+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:00:35.441+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:00:37.076+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:00:37.076+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:01:37.138+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:01:37.149+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:01:37.774+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:01:37.775+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:02:37.836+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:02:37.842+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:02:39.614+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:02:39.615+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:03:39.676+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:03:39.682+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:03:42.934+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:03:42.935+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:04:42.997+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:04:43.012+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:04:47.062+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:04:47.062+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:05:47.123+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:05:47.130+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:05:47.557+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:05:47.557+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:06:47.617+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:06:47.627+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:06:47.885+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:06:47.886+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:07:47.947+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:07:47.959+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:07:48.902+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:07:48.903+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:08:48.964+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:08:48.971+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:08:49.191+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:08:49.192+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:09:49.253+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:09:49.260+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:09:51.060+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:09:51.061+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:10:51.121+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:10:51.128+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:10:51.486+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:10:51.487+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:11:51.542+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:11:51.553+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:11:51.738+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:11:51.738+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:12:51.794+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:12:51.805+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:12:52.100+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:12:52.100+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:13:52.162+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:13:52.172+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:13:52.409+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:13:52.410+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:14:52.472+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:14:52.478+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:14:52.667+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:14:52.668+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:15:52.722+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:15:52.734+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:15:53.508+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:15:53.509+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:16:53.570+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:16:53.581+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:16:55.360+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:16:55.360+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:17:55.421+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:17:55.434+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:17:56.526+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:17:56.527+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:18:56.589+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:18:56.601+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:19:00.117+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:19:00.117+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:20:00.179+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:20:00.184+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:20:02.900+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:20:02.901+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:20:51.906+0000] {local_task_job_runner.py:302} WARNING - State of this instance has been externally set to removed. Terminating instance.
[2024-12-06T22:20:51.908+0000] {process_utils.py:131} INFO - Sending 15 to group 4074. PIDs of all processes in the group: [4074]
[2024-12-06T22:20:51.908+0000] {process_utils.py:86} INFO - Sending the signal 15 to group 4074
[2024-12-06T22:20:51.908+0000] {taskinstance.py:2450} ERROR - Received SIGTERM. Terminating subprocesses.
[2024-12-06T22:20:51.934+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=4074, status='terminated', exitcode=0, started='20:19:31') (4074) terminated with exit code 0
